images/node/Dockerfile

```diff
@@ -10,13 +10,6 @@
 #
 FROM openshift/origin
 
-COPY centos-paas-sig-openshift-origin36.repo /etc/yum.repos.d/
-COPY usr/bin/* /usr/bin/
-COPY opt/cni/bin/* /opt/cni/bin/
-COPY etc/cni/net.d/* /etc/cni/net.d/
-COPY conf/openshift-sdn-ovs.conf /usr/lib/systemd/system/origin-node.service.d/
-COPY scripts/* /usr/local/bin/
-COPY system-container/system-container-wrapper.sh /usr/local/bin/
 COPY system-container/manifest.json system-container/config.json.template system-container/service.template system-container/tmpfiles.template /exports/
```

Comment:
They should be in different RPMs, right? Not sure about OVS (also not clear why our repository is where the OVS systemd files are) but the other could just be in `origin` and `origin-node`